### PR TITLE
Clean up README

### PR DESCRIPTION
**Pull request by MattDHill**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -134,67 +134,55 @@ Enhancement suggestions are tracked as [GitHub issues](https://github.com/Start9
...
-Once you have done these things, you simply need to `cd` into the embassy-os project and then run `make agent`.
+
+First, clone the EmbassyOS repository and from the project root, pull in the submodules for dependent libraries.
+
+```
+git clone https://github.com/Start9Labs/embassy-os.git
+git submodule update --init --recursive
+```
 
 ##### ui
```
Comment (MattDHill) [inline CONTRIBUTING.md:156]: maybe just reference the ui readme here?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -134,67 +134,55 @@ Enhancement suggestions are tracked as [GitHub issues](https://github.com/Start9
...
+##### backend
   - [Install Rust](https://rustup.rs)
   - Recommended: [rust-analyzer](https://rust-analyzer.github.io/)
+  - [Docker](https://docs.docker.com/get-docker/)
+  - [Rust ARM64 Build Container](https://github.com/Start9Labs/rust-arm-builder)
+  - Scripts (run withing the `./backend` directory)
+    - `build-prod.sh` - compiles a release build of the artifacts for running on ARM64
+    - `build-dev.sh` - compiles a development build of the artifacts for running on ARM64
 
 #### Building The Image
```
Comment (MattDHill) [inline CONTRIBUTING.md:178]: So we have ui, backend, and building the image. Should these all really be under "your first code contribution"? For example, contributing to the ui does not require the backend or building the image. Same for diagnositc UI and setup-wizard. This just feels a bit disorganized.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -204,16 +192,15 @@ Contributions in the form of setup guides for integrations with external applica
 ## Styleguides
 ### Formatting
```
Comment (MattDHill) [inline CONTRIBUTING.md:193]: Maybe this should be in the UI README? Also there are different style guides for diagnostic ui, setup-wizard, and patch-db-client. Here we could say, "Each component of EmabssyOS contains its own style guide."

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -134,67 +134,55 @@ Enhancement suggestions are tracked as [GitHub issues](https://github.com/Start9
...
-- The `agent` (Haskell) is a daemon that provides the interface for the `ui` to interact with the Embassy, as well as manage system state.
-- `appmgr` (Rust) is a command line utility and (soon to be) daemon that sets up and manages services and their environments.
-
+Embassy OS has 2 main components, `backend`, and `ui`, several peripheral components.
+- The `ui` (Typescript Ionic Angular) is the code that is deployed to the browser to provide the user interface for EmbassyOS.
+- `backend` (Rust) is a command line utility, daemon, and software development kit that sets up and manages services and their environments, provides the interface for the ui, manages system state, and provides utilities for packaging services for EmbassyOS. 
+- `patch-db` - A diff based data store that is used to synchronize data between the front and backend
+- `rpc-toolkit` - A library for generating an rpc server with cli bindings from Rust functions
+- `system-images` - (Docker, Rust) A suite of utility Docker images that are preloaded with EmbassyOS to assist with functions relating to services (eg. configuration, backups, health checks)
```
Comment (kn0wmad) [inline CONTRIBUTING.md:142]: Either all points should have punctuation, or none

**Review by kn0wmad — COMMENTED** [review]
1 minor typo

**Review by elvece — APPROVED** [review]

**Pull request merged**